docs/man-openmpi/man3/MPI_Aint_add.3.rst

```diff
@@ -86,7 +86,7 @@ must correspond to locations in the same object in the same process,
 as described in MPI-3.1 section 4.1.12. The difference is calculated
 in a manner that results in the signed difference from *addr1* to
 *addr2*, as if the process that originally produced the addresses had
-called (char \*) *addr1* - (char \*) *addr2* on the addresses
+called ``(char *) addr1`` - ``(char *) addr2`` on the addresses
```

Comment:
All the fixes in this commit are tremendous.  Awesome!

FYI: we converted the man pages from wildly-inconsistent, handwritten nroff (which you can still see in the v4.1.x series) to RST via some scripting.  We thought we did a pretty good job of catching most corner cases, but you have fixed a whole boatload more corner cases that our scripting fixed.  Thank you!

Reply:
Thanks! I hope I caught the most!